frontend/src/components/custom-tools/custom-synonyms/CustomSynonyms.jsx

```diff
@@ -32,12 +32,21 @@ const columns = [
   },
 ];
 
+const PAGE_SIZE = 10;
+const SYNONYMS_LIMIT = 200;
```

Comment:
@tahierhussain It was mentioned 20 only, right?

Reply:
@Deepak-Kesavan I had a discussion with @gopalsamyperumal regarding this and we decided to go with 200 for now.